WordPress/src/main/res/layout/me_fragment.xml

```diff
@@ -214,8 +214,8 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/margin_extra_large"
-                android:paddingEnd="@dimen/margin_large"
-                android:paddingStart="@dimen/margin_large"
+                android:layout_marginEnd="@dimen/content_margin"
```

Comment:
```suggestion
                android:layout_marginBottom="@dimen/margin_extra_large"
                android:layout_marginEnd="@dimen/content_margin"
```

Should we add some margin to the bottom of the view so that the text has a little space from the bottom of the screen?  See the screenshot below for illustration.

<a href="https://user-images.githubusercontent.com/3827611/64048880-494a8500-cb30-11e9-922c-757464b9362b.png"><img src="https://user-images.githubusercontent.com/3827611/64048880-494a8500-cb30-11e9-922c-757464b9362b.png"></a>

Reply:
Good idea 👍